modules/rds/aurora-db/main.tf

```diff
@@ -82,6 +88,32 @@ resource "aws_security_group" "aurora_db" {
   }
 
   tags {
-      Name = "aurora_db"
+    Name = "aurora_db"
   }
-}
\ No newline at end of file
+}
+
+resource "aws_security_group" "aurora_db_sg" {
+  count = "${length(var.allowed_security_groups) != 0}"
+
+  name   = "aurora_db"
```

Comment:
this will barf as soon as we try to create more than one db :) 

they should be unique and just append ${var.name}

Reply:
done